roles/st2web/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+# tasks file for st2web
+
+- name: Add nginx key
+  sudo: yes
+  apt_key:
+    url: http://nginx.org/keys/nginx_signing.key
+    state: present
+
+- name: Add nginx repos
```

Comment:
Personal note: seems like all nginx-related stuff will be good to handle with external `nginx` role.
Especially more smart should be part with deleting default `/etc/nginx/sites-enabled/default` which could be pretty harmful for prod use.

Something to research/try later. For now it's good enough.